src/_premake_init.lua

```diff
@@ -268,6 +286,22 @@
 	}
 
 	api.register {
+		name = "debugtoolargs",
+		scope = "config",
+		kind = "list:string",
+		tokens = true,
+		pathVars = true,
+	}
+
+	api.register {
+		name = "debugtoolcommand",
+		scope = "config",
+		kind = "path",
+		tokens = true,
+		pathVars = true,
+	}
```

Comment:
Oh, and 'debuggerpath'... not sure.
We use 'debugcommand' to specify the binary to run, and other instances of 'path' in the api list refer to directories, not commands to execute. I don't think there's precedent for that.